.env.example

```
@@ -20,4 +20,5 @@ CONFIG_STATUS_ACTIVITY="music on {guildsCount} guilds!" # What status does the b
 CONFIG_FETCH_ALL_USERS=no # Whether to fetch all users on startup (warning: increased memory usage especially for bots with large users) (default: no) (note: You need to enable privileged GUILD_MEMBERS Intents on the developer page)
 CONFIG_SELECT_TIMEOUT=20 # Select song timeout when searching songs (in seconds) (default: 20 seconds)
 CONFIG_DISABLE_SONG_SELECTION=no # Whether to disable selecting songs on play command, this will automatically selects the first song from search results (default: no)
-CONFIG_SEARCH_MAX_RESULTS=12 # Maximum songs to list in song selection (default: 12, max: 12)
\ No newline at end of file
+CONFIG_SEARCH_MAX_RESULTS=12 # Maximum songs to list in song selection (default: 12, max: 12)
+CONFIG_YOUTUBE_SCRAPE=yes # Whether to scrape youtube instead using api. You may not fill the API config if you use this.
```

```suggestion
CONFIG_YOUTUBE_SCRAPE=yes # Whether to scrape youtube instead using API.
```
This description should be moved to YT_API_KEY's comment